operator/controllers/keptnevaluation/controller.go

```diff
@@ -138,7 +147,25 @@ func (r *KeptnEvaluationReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 				newStatus[query.Name] = evaluation.Status.EvaluationStatus[query.Name]
 				continue
 			}
-			statusItem := r.queryEvaluation(query, *evaluationProvider)
+			// resolving the SLI value
```

Comment:
pls create a function for this part, which will return the resulting statusItem

Reply:
I don't think it will improve the code. Encapsulating this would require a function with 5 parameters. IMHO this makes the code less readable and harder to follow the data-flow comparing with the current status